compiler/qsc_project/tests/projects/exclude_list/qsharp.json

```diff
@@ -0,0 +1,3 @@
+{
+  "exclude_files": [".*Excluded.qs"]
```

Comment:
I'm a little torn on one design element here. On the one hand I like that the regex here can match any part of the path, which is nice for doing the folder-based excludes you show in other tests. On the ohter hand, it seems annoying for the user who just wants to have a flat list of excluded files to have to remember to prepend each file name with `.*` to match the first part of the path. Is there a resonable way we could include the wildcard automatically to lessen the burden for users while still getting the behavior we want?

Reply:
After discussing this some more and realizing even the `.` in `.qs` needs to be escaped, there isn't an easy way to have one field pull double duty. Instead, let's split this up into two fields, one that takes only file names and one that takes regex. That way the simple and advanced cases are both covered without stepping on each other's toes.